crypto/stark-verifier-ethereum/contracts/testing_contracts/prime_field_tester.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity ^0.6.4;
+
+import '../primefield.sol';
+
+contract PrimeFieldTester is PrimeField {
```

Comment:
Can we generate this wrapper using our tools? <https://github.com/0xProject/0x-monorepo/blob/ad3b0b0ce99bc33baffab80e31504f6120ddf2a9/packages/sol-meta/README.md>

Reply:
This might be possible, but integrating 0x tooling into the buidler setup is not obviously trivial so should be implemented in it's own pr imo

Reply:
for sure!